.gitignore

```diff
@@ -130,3 +130,6 @@ dmypy.json
 
 # JUnit
 junit.xml
+
+# VSCode
+.vscode/
```

Comment:
There is `insert_final_newline = true` violation from the [`.editorconfig` file](https://github.com/9dogs/tg-odesli-bot/blob/156fc40/.editorconfig).

Reply:
Also I think it should be done in a separate PR.